src/sentry/digests/notifications.py

```diff
@@ -19,10 +20,20 @@
 
 
 def split_key(key):
-    from sentry.plugins.base import plugins
+    from sentry.mail.adapter import ActionTargetType
+
+    key_parts = key.split(":", 4)
+    project_id = key_parts[2]
+    if len(key_parts) == 5:
```

Comment:
Kinda unfortunate we have to do this. Maybe leave a comment about the two types of formats?

Reply:
We'll get rid of this eventually, we just need to support both formats as a transition step.